client/landing/jetpack-cloud/sections/backups/rewind-flow/download.tsx

```diff
@@ -0,0 +1,190 @@
+	const dispatch = useDispatch();
+	const moment = useLocalizedMoment();
+	const translate = useTranslate();
+
+	const [ rewindConfig, setRewindConfig ] = useState< RewindConfig >( defaultRewindConfig );
+
+	const downloadId = useSelector( state => getBackupDownloadId( state, siteId, rewindId ) );
+	const downloadUrl = useSelector( state => getBackupDownloadUrl( state, siteId, rewindId ) );
+	const downloadProgress = useSelector( state =>
+		getBackupDownloadProgress( state, siteId, rewindId )
+	);
+
+	const downloadTimestamp = moment.unix( rewindId ).format( 'LLL' );
+
+	const requestDownload = useCallback(
+		() => dispatch( rewindBackup( siteId, rewindId, rewindConfig ) ),
+		[ dispatch, rewindConfig, rewindId, siteId ]
+	);
+
+	const renderConfirm = () => (
+		<>
+			<div className="rewind-flow__header">
+				<img
+					src="/calypso/images/illustrations/jetpack-cloud-download-ready.svg"
+					alt="jetpack cloud download ready"
+				/>
+			</div>
+			<h3 className="rewind-flow__title">{ translate( 'Create downloadable backup' ) }</h3>
+			<p className="rewind-flow__info">
+				{ translate(
+					'{{strong}}%(downloadTimestamp)s{{/strong}} is the selected point to create a download backup of. ',
+					{
+						args: {
+							downloadTimestamp,
+						},
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<h4 className="rewind-flow__cta">{ translate( 'Choose the items you wish to restore:' ) }</h4>
+			<RewindConfigEditor currentConfig={ rewindConfig } onConfigChange={ setRewindConfig } />
+			<RewindFlowNotice
+				gridicon="notice-outline"
+				title={ translate( 'More info' ) }
+				type={ RewindFlowNoticeLevel.NOTICE }
+				link="https://jetpack.com/support/backup"
+			/>
+			<Button
+				className="rewind-flow__primary-button"
+				primary
+				onClick={ requestDownload }
+				disabled={ Object.values( rewindConfig ).every( setting => ! setting ) }
+			>
+				{ translate( 'Create downloadable file' ) }
+			</Button>
+		</>
+	);
+
+	const renderInProgress = () => (
+		<>
+			<div className="rewind-flow__header">
+				<img
+					src="/calypso/images/illustrations/jetpack-cloud-download-ready.svg"
+					alt="jetpack cloud download ready"
+				/>
+			</div>
+			<h3 className="rewind-flow__title">
+				{ translate( 'Currently creating a downloadable backup of your site' ) }
+			</h3>
+			<ProgressBar percent={ downloadProgress } />
+			<p className="rewind-flow__info">
+				{ translate(
+					"We're creating a downloadable backup of your site from {{strong}}%(downloadTimestamp)s{{/strong}}.",
+					{
+						args: {
+							downloadTimestamp,
+						},
+						components: {
+							strong: <strong />,
+						},
+					}
+				) }
+			</p>
+			<CheckYourEmail
+				message={ translate( "For your convenience, we'll email you when your file is ready." ) }
+			/>
+		</>
+	);
+
+	const renderReady = () => (
+		<>
+			<div className="rewind-flow__header">
```

Comment:
This div seems to be repeated quite a few times. 
Can we make it a component?

Reply:
I was waiting until I had both flows to make a final decision on if these four line elements needed their own components